style.css

```diff
@@ -189,7 +189,7 @@ ul {
   background-color: #36b37e;
   border-radius: 5px;
   border-style: none;
-  color: white;
+  color: black;
   font-size: 17px;
```

Comment:
**Contrast ratio ("color contrast")**:

* The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.
Please check the report [HERE](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2022-02-22-at-20-27-40-935?failing) according to the project requirement.

Reply:
I changed the color from white to black. The black color increased the contrast ratio thus making the website accessibility to  100%.